template.distillery/Makefile.db

```diff
@@ -30,8 +30,20 @@ ELIOMDEP          := $(ENV_PSQL) $(ELIOMDEP)
 # should be the same as ENV_PSQL)
 OCSIGENSERVER     := $(OCSIGENSERVER)
 OCSIGENSERVER.OPT := $(OCSIGENSERVER.OPT)
+
 ## ---------------------------------------------------------------------
 
+##----------------------------------------------------------------------
+
+pg_dump = pg_dump
+
+pg_ctl  = $(shell \
+           sh -c "which pg_ctl" || \
+           sh -c "which pg_ctlcluster" || \
+           sh -c "if [ -f pg_ctl ]; then (echo ./pg_ctl) else (echo '') fi")
```

Comment:
I proposed in #123 to set `pg_cltcluster` first but it's not a good idea because on Ubuntu, it doesn't work for me. The executable `pg_ctlcluster` is located in `/usr/bin/pg_ctlcluster` and it's not working.

```
$> make db-init
Makefile.mobile:242: warning: overriding recipe for target 'cordova/www/css/snapshot.css'
Makefile.mobile:156: warning: ignoring old recipe for target 'cordova/www/css/snapshot.css'
mkdir -p local_db
/usr/bin/pg_ctlcluster initdb -D local_db
Unknown option: d
Makefile.db:98: recipe for target 'db-init' failed
make: *** [db-init] Error 1
```

(just a few minutes for Debian, I got some issue with my Debian server and OPAM).